release.sh

```diff
@@ -145,21 +153,43 @@ if [ $dodist -eq 0 ]; then
     # We haven't tagged yet, so tell the dist script what version
     # it's building
     DIST_VERSION="$tag" npm run dist
+
+    `dirname $0`/scripts/changelog_head.py > latest_changes.md
```

Comment:
are we doing anything here to ensure that the riot-web changelog references or links to the equivalent matrix-react-sdk changelog (and similarly js-sdk changelog)?  At the least, we might want to include non-versioned links to the latest_changes for those projects.  And slightly better would be links to the right version of latest_changes for the deps...